log_outgoing_requests/log_requests.py

```diff
@@ -20,16 +20,16 @@ def install_outgoing_requests_logging():
     Log all outgoing requests which are made by the library requests during a session.
     """
 
-    if hasattr(Session, "_original_request"):
+    if hasattr(Session, "_initial_request"):
```

Comment:
let's give this a proper namespaced name

```suggestion
    if hasattr(Session, "_dlor_original_request"):
```